src/config/governence.json

```diff
@@ -6,18 +6,44 @@
   },
   "permissive": {
     "connections": {
-      "invitation": "always",
-      "request": "always"
+      "create-invitation": "always",
```

Comment:
I don't know what to even put here any more.   The design no longer fits use case.